formatting-code/a.kraychev/JavaScript-code.js

```diff
@@ -0,0 +1,126 @@
+b = navigator.appName;
+addScroll = false;
+
+if (navigator.userAgent.indexOf('MSIE 5') > 0
+    || navigator.userAgent.indexOf('MSIE 6') > 0
+) {
+  addScroll = true;
+}
+
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+document.onmousemove = mouseMove;

```

Comment:
This should be either in separate paragraph or in one paragraph with the next if.